### ISSUE TITLE
GDExtension, the new GDNative (or why did my Godot 3 plugin just break?!?)

### ISSUE THREAD

**Issue by BastiaanOlij** [godotengine/godot-cpp#623]
Unfortunately there is no good way to do a headsup that reaches everyone so if you're one of the people who just pulled master and saw their project stop working, we are sorry!

However don't panic, all you need to do is switch to the `3.x` branch of this repository and all should be right with the world once again!

With that out of the way, we did a wonderful [blog post with all the ins and outs of what's going on](https://godotengine.org/article/introducing-gd-extensions).

The TL;DR is, the `master` branch of this repository is now for Godot 4 development, while the `3.x` branch is for Godot 3 development, that simple :)

**Comment by akien-mga** [comment]
I guess we can unpin and close this now :)
